build.gradle

```diff
@@ -67,5 +67,6 @@ subprojects {
 
     tasks.withType(JavaCompile) {
         options.compilerArgs += ['-Xlint:deprecation', '-Werror']
+        options.errorprone.errorproneArgs += '-Xep:Finally:OFF'
```

Comment:
I actually just tried this on a much larger internal project (SMM) and errorprone passed nicely when I turned off this single rule!